redis-wrapper/src/main/java/redis/RedisPipeline.java

```diff
@@ -0,0 +1,65 @@
+package redis;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pluggable.Result;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.Pipeline;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RedisPipeline implements pluggable.Pipeline {
+    private Pipeline _pipelined;
+    private final String _prefix;
+    private final JedisPool _jedisPool;
+
+    private static final Logger _log = LoggerFactory.getLogger(RedisPipeline.class);
+
+    public RedisPipeline(JedisPool jedisPool, String prefix) {
+        _jedisPool = jedisPool;
+        _prefix = prefix;
+        try (Jedis jedis = _jedisPool.getResource()) {
+            _pipelined = jedis.pipelined();
+        } catch (Exception ex) {
+            new RedisException(ex.getMessage());
+        }
+    }
+
+    /* package private */ String buildKeyWithPrefix(String key) {
+        if (!key.startsWith(_prefix)) {
```

Comment:
this checking is an extra thing that should be removed, if for some reason two prefix are concatenated, definitely we have a bug in the code

Reply:
The same before comment.